osu.Game/Beatmaps/WorkingBeatmap.cs

```diff
@@ -327,7 +327,15 @@ public Track Track
         public bool SkinLoaded => skin.IsResultAvailable;
         public ISkin Skin => skin.Value;
 
-        protected abstract ISkin GetSkin();
+        /// <summary>
+        /// Creates a new skin instance for this beatmap.
+        /// </summary>
+        /// <remarks>
+        /// This should only be called externally in scenarios where it is explicitly desired to get a new instance of a skin
```

Comment:
Fine to say this, but it's weird that it's only mentioned on this method and not the other `Get` methods (I believe they are all equally dangerous).